### PR TITLE
Feature/15 endpoint to retrive list of books

### PR DESCRIPTION
**Pull request by alinschuller**
TBD

### CODE REVIEW

**Comment by alinschuller** [comment]
[Endpoint to retrive list of books](https://trello.com/c/7KttMP8W/15-endpoint-to-retrive-list-of-books)

File: BigriverBookstore-api/BigriverBookstore-api.Server.WebService/Configurations/BookConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                .SetApiCollectionPathSegment("book");
+            
+            this.ResourceIdentity(x => x.BookId)
+                .SetApiType("books");
+
+            this.Attribute(x => x.Title)
+                .SetApiPropertyName("book-title");
+
+            this.Attribute(x => x.DatePublished)
+                .SetApiPropertyName("date-published");
```
Comment (pirvudoru) [inline BigriverBookstore-api/BigriverBookstore-api.Server.WebService/Configurations/BookConfiguration.cs]: Why did you choose this naming convention (with dashes) for the exported attribute names?

Comment (alinschuller) [reply BigriverBookstore-api/BigriverBookstore-api.Server.WebService/Configurations/BookConfiguration.cs]: I based this naming convention on the documentation found from JsonApiFramework, but I could also remove the dashes and use uppercase for the first letter of the second word or use underline between the words.

Comment (alinschuller) [reply BigriverBookstore-api/BigriverBookstore-api.Server.WebService/Configurations/BookConfiguration.cs]: For me it was important to have a clear reading of the property

Comment (alinschuller) [reply BigriverBookstore-api/BigriverBookstore-api.Server.WebService/Configurations/BookConfiguration.cs]: Updated properties names to Camel Case.

File: BigriverBookstore-api/BigriverBookstore-api.Server.WebService/BookRepository.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+using System.Collections.Generic;
+using System.Linq;
+
+using BigriverBookstore_api.ServiceModel;
+
+
+namespace BigriverBookstore_api.WebService
+{
+
+    public static class BookRepository
```
Comment (alexandru-calinoiu) [inline BigriverBookstore-api/BigriverBookstore-api.Server.WebService/BookRepository.cs:11]: In order to be consistent with the directory structure choose I would though that repositories should have been namespaced under `Repositories`, same as `Controllers` and `Configurations`

Not that this is the best way to separate them, I would argue that separating by features is better then by concepts.

**Review by alexandru-calinoiu — COMMENTED** [review]
Apart from the comment that I've put this pr has no tests.

**Pull request closed without merging**